test/__tests__/dangling-commas-test.js

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
I just realized all the tests have this, but this is not a facebook project. I'm going to remove this from all other tests, would you mind removing this header here? :) thanks!